pep/translations/es-es.json

```diff
@@ -67,6 +67,13 @@
             "label": "Sort:",
             "year": "Year"
         },
+        "item": {
```

Comment:
I would recommend taking these out so we still get warnings. Otherwise your temporarily solving a build issue that will turn into a runtime issues later on

Reply:
not sure i follow. the spanish translations file was just created to test the i10n/i18n switching stuff early on, and I've been keeping it in sync w/the english file, just duplicating the english translations for now, as it will ultimately need the exact same keys duplicated (probably starting off w/the english strings), and then handed off to the PEP people to translate.

I can just disable spanish in the app for now and then just delete this file until we are ready to translate, rather than seeing these housekeeping commits as we go along, if we prefer.

Reply:
I believe I have observed build errors when you have a key that's in the English file, that's not in the Spanish one. I was leaving those as it acted as a reminder that we need to get them translated. If you want to disable Spanish and delete the file for now, that's fine too as long as we just have a story to remind ourselves to get all the translations

Reply:
yea theres a separate story/release altogether to actually enabling language switching/translations (I believe its not even for the first release, Rel.2 i think).

But yeah, I think the intention is to hand them the json files that contain all the keys/default english translations, so that they can then convert and translate to each language. So having the build warnings for missing translations is just unnecessary noise for us, since we aren't going to be responsible for actually putting in translations ourselves as we add new stuff for the initial release.